FSEvents/FSEvents_globals.ml

```diff
@@ -0,0 +1,171 @@
+(* auto-generated, do not modify *)
+
+open CoreFoundation
+open Runtime
+open Objc
+
+module FSEventStream = struct
+  let t : [ `FSEventStream ] structure typ = structure "FSEventStream"
+end
+
+module ConstFSEventStream = struct
+  let t : [ `ConstFSEventStream ] structure typ = structure "ConstFSEventStream"
+end
+
+module ConstFSEventStreamRef = struct
+  let t = ptr FSEventStream.t
+end
+
+module FSEventStreamRef = struct
+  let t = ptr FSEventStream.t
+end
+
+let kFSEventStreamCreateFlagFileEvents = 16
+let kFSEventStreamCreateFlagFullHistory = 128
+let kFSEventStreamCreateFlagIgnoreSelf = 8
+let kFSEventStreamCreateFlagMarkSelf = 32
+let kFSEventStreamCreateFlagNoDefer = 2
+let kFSEventStreamCreateFlagNone = 0
+let kFSEventStreamCreateFlagUseCFTypes = 1
+let kFSEventStreamCreateFlagUseExtendedData = 64
+let kFSEventStreamCreateFlagWatchRoot = 4
+let kFSEventStreamCreateWithDocID = 256
+let kFSEventStreamEventFlagEventIdsWrapped = 8
+let kFSEventStreamEventFlagHistoryDone = 16
+let kFSEventStreamEventFlagItemChangeOwner = 16384
+let kFSEventStreamEventFlagItemCloned = 4194304
+let kFSEventStreamEventFlagItemCreated = 256
+let kFSEventStreamEventFlagItemFinderInfoMod = 8192
+let kFSEventStreamEventFlagItemInodeMetaMod = 1024
+let kFSEventStreamEventFlagItemIsDir = 131072
+let kFSEventStreamEventFlagItemIsFile = 65536
+let kFSEventStreamEventFlagItemIsHardlink = 1048576
+let kFSEventStreamEventFlagItemIsLastHardlink = 2097152
+let kFSEventStreamEventFlagItemIsSymlink = 262144
+let kFSEventStreamEventFlagItemModified = 4096
+let kFSEventStreamEventFlagItemRemoved = 512
+let kFSEventStreamEventFlagItemRenamed = 2048
+let kFSEventStreamEventFlagItemXattrMod = 32768
+let kFSEventStreamEventFlagKernelDropped = 4
+let kFSEventStreamEventFlagMount = 64
+let kFSEventStreamEventFlagMustScanSubDirs = 1
+let kFSEventStreamEventFlagNone = 0
+let kFSEventStreamEventFlagOwnEvent = 524288
+let kFSEventStreamEventFlagRootChanged = 32
+let kFSEventStreamEventFlagUnmount = 128
+let kFSEventStreamEventFlagUserDropped = 2
+let kFSEventStreamEventIdSinceNow = -1
+
+let _FSEventStreamCopyDescription =
+  Foreign.foreign "FSEventStreamCopyDescription"
+    (ptr FSEventStream.t @-> returning (ptr CFString.t))
+
+let _FSEventStreamCopyPathsBeingWatched =
+  Foreign.foreign "FSEventStreamCopyPathsBeingWatched"
+    (ptr FSEventStream.t @-> returning (ptr CFArray.t))
+
+let _FSEventStreamCreate =
+  Foreign.foreign "FSEventStreamCreate"
+    (ptr CFAllocator.t
+    @-> ptr (ptr void)
```

Comment:
This seems wrong: https://developer.apple.com/documentation/coreservices/1443980-fseventstreamcreate 

Seems that type aliases are not working properly?